xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -597,9 +597,16 @@ int PlayOrQueueMedia(const std::vector<std::string>& params, bool forcePlay)
     }
 
     if ((item.IsAudio() || item.IsVideo()) && !item.IsSmartPlayList())
+    {
+      if (!item.HasProperty("playlist_type_hint"))
+        item.SetProperty("playlist_type_hint", PLAYLIST::TYPE_MUSIC);
```

Comment:
@CastagnaIT does this make sense, just wondering? 
You enter the if the item is either audio and video and set it to music.

Reply:
this is just a hint is only used when the type is not detected (fallback),
it is a behaviour that was already there in source code but in a different form, only now it is more evident thanks to the "hint"

at least now imo it is more clear that further improvements should be made in the future,
forcing on the 'music' type is already present also on other places of the source code as well, e.g. file browser

I agree this is not quite right to force on 'music', but atm there is no better solution,
a rework on how is implemented playlists is needed to remove this kind of fallback